deploy/spoke/clusterrole.yaml

```diff
@@ -0,0 +1,36 @@
+# Clusterrole for work agent in addition to admin clusterrole.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: open-cluster-management:work:agent
+rules:
+# Allow agent to get/list/watch/create/delete crds.
+- apiGroups: ["apiextensions.k8s.io"]
+  resources: ["customresourcedefinitions"]
+  verbs: ["get", "list", "watch", "create", "delete", "update"]
+# Allow agent to create/delete namespaces, get/list are contained in admin role already
+- apiGroups: [""]
+  resources: ["namespaces"]
+  verbs: ["create", "delete"]
+# Allow agent to manage role/rolebinding/clusterrole/clusterrolebinding
+- apiGroups: ["rbac.authorization.k8s.io"]
+  resources: ["clusterrolebindings", "rolebindings"]
+  verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
+- apiGroups: ["rbac.authorization.k8s.io"]
+  resources: ["clusterroles", "roles"]
+  verbs: ["get", "list", "watch", "create", "update", "patch", "delete", "escalate", "bind"]
+# Allow agent to create sar
+- apiGroups: ["authorization.k8s.io"]
+  resources: ["subjectaccessreviews"]
+  verbs: ["create"]
+# Allow agent to create events
+- apiGroups: ["", "events.k8s.io"]
+  resources: ["events"]
+  verbs: ["create", "patch", "update"]
+# Allow agent to managed appliedmanifestworks
+- apiGroups: ["work.open-cluster-management.io"]
+  resources: ["appliedmanifestworks"]
+  verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
+- apiGroups: ["work.open-cluster-management.io"]
```

Comment:
see https://github.com/open-cluster-management/registration-operator/pull/49/files#diff-15835a369867a0527a71590c6693ae68R468-R470 . I think you need update on appliedmanifestworks/finalize too.

Reply:
Fixed